build.js

```diff
@@ -293,11 +293,7 @@ function getSource (callback) {
           text: '#BlackLivesMatter',
           link: '/en/black-lives-matter/'
         },
-        banner: {
-          visible: true,
-          text: 'New security releases now available for 16.x, 14.x, and 12.x release lines',
-          link: '/en/blog/vulnerability/july-2021-security-releases/'
-        }
+        banner: i18nJSON().banner
```

Comment:
You might be able to drop this entirely and do it directly in the template https://github.com/nodejs/nodejs.org/blob/b6b02407d5f3174513152bcedb11ab3ee4108a2e/layouts/index.hbs#L21

Reply:
Good call, you can def use it there. Since the `blacklivesmatter` object has a boolean property also, should I just pull that into the `site.json` and do the same thing?

Reply:
Yup, I think doing the same would make sense if you want to do that here too